include/deal.II/lac/la_parallel_vector.h

```diff
@@ -185,25 +185,31 @@ namespace LinearAlgebra
      * fail in some circumstances. Therefore, it is strongly recommended to
      * not rely on this class to automatically detect the unsupported case.
      *
-     * <h4>CUDA support</h4>
+     * <h4>GPU support</h4>
      *
-     * This vector class supports two different memory spaces: Host and CUDA. By
-     * default, the memory space is Host and all the data are allocated on the
-     * CPU. When the memory space is CUDA, all the data is allocated on the GPU.
-     * The operations on the vector are performed on the chosen memory space. *
-     * From the host, there are two methods to access the elements of the Vector
-     * when using the CUDA memory space:
+     * This vector class supports two different memory spaces: Host and Default.
+     * By default, the memory space is Host and all the data is allocated on the
```

Comment:
Maybe says something like `if the template parameter is not given` instead of default. Right it sounds a little bit like the Default memory space is Host

Reply:
Done.